plugin/al/plugin/AL_USDMayaTestPlugin/py/testLayerManager.py

```diff
@@ -17,37 +17,50 @@
 #
 
 import os
-import tempfile
+import shutil
 import unittest
 
 from maya import cmds
 from AL.usdmaya import ProxyShape
 
-import fixturesUtils
```

Comment:
We use `fixturesUtils` all over the place. What was the reason to remove it from AL tests?

Reply:
Internally we have not set up our testing to take advantage of the module, however that transition is underway and a PR will be up for it at some point. I'll also be removing tempfile usage across AL tests.

Reply:
That’s great to hear. Looking forward to this PR.

Reply:
@J-Mo63 Looks like I didn't notice something in your change. All our tests are writing files to the build folder. I just run the new tests locally and you write the source folder...this means that now I always have changes in the source tree after running tests, e.g. 
```
git status
HEAD detached at origin/dev
Untracked files:
  (use "git add <file>..." to include in what will be committed)
	plugin/al/plugin/AL_USDMayaTestPlugin/py/testLayerManagerOutput/
```
 Please adjust it to match what other tests are doing.

Reply:
Ah yes that's no good at all. The switch to `fixturesUtils` is on the way.